app/src/lib/databases/repositories-database.ts

```diff
@@ -23,6 +23,11 @@ export interface IDatabaseGitHubRepository {
   readonly lastPruneDate: number | null
 }
 
+export interface IDatabaseProtectedBranch {
```

Comment:
Can we add a tiny bit of documentation to explain what this does.

Reply:
Added in 3581f36e87387ae4fa8beba20ab9a0b0d90c630c